public/app/plugins/panel/canvas/utils.ts

```diff
@@ -143,6 +143,14 @@ export function getDataLinks(
 
   const elementConfig = elementOptions.config;
 
+  elementOptions.links?.forEach((link) => {
+    const key = `${link.title}/${link.href}`;
+    if (!linkLookup.has(key)) {
+      links.push(link);
+      linkLookup.add(key);
```

Comment:
It seems like the links being added here from the element options aren't actually clickable for some reason 🤔 

 

https://github.com/grafana/grafana/assets/22381771/8fa64702-6c5f-4c69-866f-8a139b238823